src/olympia/reviewers/tests/test_utils.py

```diff
@@ -1690,6 +1689,79 @@ def test_reject_multiple_versions_content_review(self):
         assert self.check_log_count(amo.LOG.REJECT_VERSION.id) == 0
         assert self.check_log_count(amo.LOG.REJECT_CONTENT.id) == 2
 
+        # Check points awarded.
+        self._check_score(amo.REVIEWED_CONTENT_REVIEW)
+
+    def test_reject_multiple_versions_content_review_with_delay(self):
+        self.grant_permission(self.request.user, 'Addons:ContentReview')
+        old_version = self.version
+        self.version = version_factory(addon=self.addon, version='3.0')
+        self.setup_data(
+            amo.STATUS_APPROVED, file_status=amo.STATUS_APPROVED,
+            content_review=True)
+
+        # Pre-subscribe the user to new listed versions of this add-on, it
+        # shouldn't matter.
+        ReviewerSubscription.objects.create(
+            addon=self.addon, user=self.request.user)
+
+        in_the_future = datetime.now() + timedelta(days=14)
+
+        # Safeguards.
+        assert isinstance(self.helper.handler, ReviewFiles)
+        assert self.addon.status == amo.STATUS_APPROVED
+        assert self.file.status == amo.STATUS_APPROVED
+        assert self.addon.current_version.is_public()
+
+        data = self.get_data().copy()
+        data.update({
+            'versions': self.addon.versions.all(),
+            'delayed_rejection': True,
+            'delayed_rejection_days': 14,
+        })
+        self.helper.set_data(data)
+        self.helper.handler.reject_multiple_versions()
+
+        # File/addon status didn't change.
+        self.addon.reload()
+        self.file.reload()
+        assert self.addon.status == amo.STATUS_APPROVED
+        assert self.addon.current_version == self.version
+        assert list(self.addon.versions.all()) == [self.version, old_version]
+        assert self.file.status == amo.STATUS_APPROVED
+
+        # The versions are now pending rejection.
+        for version in self.addon.versions.all():
+            assert version.pending_rejection
+            self.assertCloseToNow(version.pending_rejection, now=in_the_future)
+
+        assert len(mail.outbox) == 1
+        assert mail.outbox[0].to == [self.addon.authors.all()[0].email]
```

Comment:
Nit: Same comment as above about creating a shortcut to `mail.outbox[0]`.

Reply:
9343bbd66aeda3c992c09d7093b9eca89a041086